core_lib/src/structure/layer.h

```diff
@@ -170,6 +170,8 @@ class Layer
     /** Clear the list of dirty keyframes */
     void clearDirtyFrames() { mDirtyFrames.clear(); }
 
+    std::map<int, KeyFrame*, std::greater<int>> keyframes() const { return mKeyFrames; }
```

Comment:
Could you please remove this unused function?

I don't want to expose the internal data structure outside the Layer class. Looking back, my decision to use std::map instead of std::vector wasn't very smart in hindsight.

Reply:
Done

Reply:
thanks!